ts/src/header-validator/trigger.test.ts

```diff
@@ -1099,6 +1100,50 @@ const testCases: jsontest.TestCase<Trigger>[] = [
     ],
   },
 
+  // Attribution Scope
+  {
+    name: 'attribution-scope-not-string',
+    json: `{"attribution_scopes": [1, 2]}`,
+    expectedErrors: [
+      {
+        path: ['attribution_scopes', 0],
+        msg: 'must be a string',
+      },
+      {
+        path: ['attribution_scopes', 1],
+        msg: 'must be a string',
+      },
+    ],
+  },
+  {
+    name: 'attribution-scopes-empty-list',
+    json: `{
+      "attribution_scopes": []
```

Comment:
This is supposed to be an error, isn't it?

Reply:
This should be valid case now as we treat empty list the same as null per your suggestion in the explainer.

Reply:
If the empty list is valid, we shouldn't be returning `None` in the header validator.